docs/pages/deployment/pex.rst

```diff
@@ -22,71 +22,84 @@ To use file-based configuration, you need to define the path to a directory that
 
-File-based configuration
-************************
+Policy Structure
+****************
 
-JSON files used for file-based configuration must have the following structure:
+JSON documents used for policies must have the following structure:
 
 .. code-block:: json
 
-	{
-		"example_scope": {
-			"organization": {
-				"id": "example",
-				"format": {
-					"ldp_vc": {
-						"proof_type": ["JsonWebSignature2020"]
-					},
-					"ldp_vp": {
-						"proof_type": ["JsonWebSignature2020"]
-					}
-				},
-				"definition": {
-					"input_descriptors": [
-						{
-							"id": "1",
-							"constraints": {
-								"fields": [
-									{
-										"path": ["$.type"],
-										"filter": {
-											"type": "string",
-											"const": "ExampleCredential"
-										}
-									}
-								]
-							}
-						}
-					]
-				}
-			}
-		}
-	}
-
-Where `example_scope` is the scope that the presentation definition is associated with.
-The `presentation_definition` object contains the presentation definition that should be used for the given scope.
-The `wallet_owner_type` field is used to determine the audience type of the presentation definition, valid values are `organization` and `user`.
+  {
+    "example_scope": {
+      "organization": {
+        "id": "example",
+        "format": {
+          "ldp_vc": {
+            "proof_type": ["JsonWebSignature2020"]
+          },
+          "ldp_vp": {
+            "proof_type": ["JsonWebSignature2020"]
+          }
+        },
+        "definition": {
+          "input_descriptors": [
+            {
+              "id": "1",
+              "constraints": {
+                "fields": [
+                  {
+                    "id": "example_credential_type",
+                    "path": ["$.type"],
+                    "filter": {
+                      "type": "string",
+                      "const": "ExampleCredential"
+                    }
+                  }
+                ]
+              }
+            }
+          ]
+        }
+      }
+    }
+  }
+
+Where ``example_scope`` is the scope that the presentation definition is associated with.
+The ``presentation_definition`` object contains the presentation definition that should be used for the given scope.
+The ``wallet_owner_type`` field is used to determine the audience type of the presentation definition, valid values are ``organization`` and ``user``.
+
+OAuth2 Token Introspection field mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The input descriptor constraint fields that contain an ``id`` property (``input_descriptor.contraints.field.id``) are returned in the OAuth2 Token Introspection response.
+The value of the Verifiable Credential that the matched field is included in the response as claims.
+E.g., in the example above, a claim named ``example_credential_type`` is added to the introspection response, containing the Verifiable Credential ``type`` property.
```

Comment:
You mean the value of the `type` property. An example of an introspection response would be really helpful here.